eth/stagedsync/stage_blockhashes.go

```diff
@@ -61,6 +61,11 @@ func SpawnBlockHashStage(s *StageState, tx kv.RwTx, cfg BlockHashesCfg, ctx cont
 		return nil
 	}
 
+	// Move 100 block at a time, required for witness generation stage
+	if headNumber-s.BlockNumber > 100 {
+		headNumber = s.BlockNumber + 100
```

Comment:
It would be great if you can explain why we need to move 100 block at a time for witness stage. Also, is it possible not to use a hard coded number?